README.md

```diff
@@ -77,9 +77,11 @@ Detectors are configured via the `detectors` configuration value. Right now, the
 Setting Up the Notifiers
 ------------------------
 
-If a secret is detected, we will notify you using your configured `notifiers`. Currently, the two notification methods are printing to the console, and notifying via Slack.
+If a secret is detected, we will notify you using your configured `notifiers`. Currently, the two notification methods are printing to the console, and notifying via a Slack/Teams webhook. 
 
-To configure Slack notifications, create the following configuration option with the `webhook_url` provided by Slack:
+For webhook notifications, both Slack and Microsoft Teams implementations work identically: the JSON structure used to call the Slack webhook is the same as for Microsoft Teams, the implementation is just currently not updated to reflect this. To configure Teams notifications, follow these same steps but with a Teams Connector.
+
+To configure Slack/Teams notifications, create the following configuration option with the `webhook_url` provided by Slack:
```

Comment:
Only change here could be to reference Slack/Teams in the last mention of Slack. I might add some clarifying text to explain that this was a happy accident, but that's something Jordan or I can take on at a later date. Thank you for the PR!